projects/uitdatabank/models/ownership-post.json

```diff
@@ -47,5 +11,27 @@
       "itemType": "organizer",
       "ownerEmail": "foo@bar.com"
     }
-  ]
-}
+  ],
+  "type": "object",
+  "properties": {
+    "itemId": {
+      "type": "string",
+      "x-stoplight": {
+        "id": "5zhh7adp7ig7k"
+      },
+      "description": "The id of item of which you request ownership."
```

Comment:
We have some legacy items which have malformed UUID's therefore we cannot use the UUID type.

Reply:
It's also for Organizers => https://io.uitdatabank.be/organizers/29DD97B4-06D5-6C4D-6025C1D5485CC083
But we also use the custom regex in Stoplight, so it's not a problem.